e2e/wdio/core/tests/tabs.e2e-spec.ts

```diff
@@ -0,0 +1,154 @@
+import {
+    click,
+    doubleClick,
+    getAttributeByName,
+    getElementArrayLength,
+    getElementClass,
+    getText,
+    refreshPage,
+    setValue,
+    waitForPresent,
+} from '../../driver/wdio';
+import { tabsPo } from '../../core/pages/tabs.po';
+
+describe('Tabs test suite', () => {
+
+    const tabsPage = new tabsPo();
+    const {
+        AddExample, tabsExample, FilterExample, ProcessExample, IconOnlyExample, SelectionExample,
+        collapsibleExample, stackendContentExample, collapsibleOverflowExample,
+        fdTab, addBtn, removeBtn, chooseTabsBtn, expandedListPoint, moreBtn, tabPanel,
+        modeSelect, iconOnlyMode, compactCheckBox, threeElementsRow, titleField,
+        counterField, contentField, icon1, titleAndCountSection, contentSection, collapsibleTab, acceleratedIcon,
+        fdIcon, filterMode
+    } = tabsPage;
+
+    beforeAll(() => {
+        tabsPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(fdTab);
+    }, 1);
+
+    it('check tabs select', () => {
+        expect(checkTabsSelect(tabsExample)).toBe(true);
```

Comment:
when we have assertions that we expect to equal a boolean value (true/false), we add an on error message. looks like this:
expect(checkTabsSelect(tabsExample)).toBe(true, 'tab is not selected');

please apply to relevant cases